riscv/insns/ssrdp.h

```diff
@@ -0,0 +1,5 @@
+require_extension(EXT_ZICFISS);
+
+reg_t ssp = STATE.ssp->read();
+
```

Comment:
Need to qualify by xSSE. When xSSE is 0, the instruction reverts to being a zimop.

Reply:
Add check implementation